android/src/main/kotlin/com/example/android_locator_plugin/LocationHelper.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.android_locator_plugin
+
+import com.google.android.gms.location.LocationResult
+import kotlin.math.pow
+import kotlin.math.round
+
+fun LocationResult.convertToString(): String {
+    return StringBuilder().append(AndroidLocatorPlugin.LONGITUDE)
+        .append(Constants.WHITE_SPACE)
+        .append(this.lastLocation?.longitude?.run {
+            this.cutToDecimals(Constants.DECIMALS_NUMBER)
+        })
+        .append(Constants.WHITE_SPACE)
+        .append(AndroidLocatorPlugin.LATITUDE)
+        .append(Constants.WHITE_SPACE)
+        .append(this.lastLocation?.latitude?.run {
+            this.cutToDecimals(Constants.DECIMALS_NUMBER)
+        }).toString()
+}
+
+fun Double.cutToDecimals(decimals: Int): Double {
+    val numbOfDigits = Constants.BASE.toDouble().pow(decimals)
+    return round(this * numbOfDigits) / numbOfDigits
+}
```

Comment:
you missed the end of file extra line. This can be configured on Android Studio, check this post to do it https://stackoverflow.com/questions/24093609/how-can-i-configure-android-studio-to-add-new-line-at-end-of-file/28660298#28660298

Reply:
added and configured, thank you